src/components/SideNav/Provider.js

```diff
@@ -24,28 +24,31 @@ export const Provider = ({ children }) => {
   );
 
   React.useEffect(() => {
+    const contentDom = document.querySelector("#content-column");
```

Comment:
I thought about using `useRef` in `Documentation.js` or `ApiReference.js` instead of `document.querySelector`, but am not sure about passing the ref around. Let me know.

Reply:
I think this is solid. There have been several spots now where, after getting it working, I realized I probably could have done some query selectors and simplified the end result, passing refs around has a lot of weird plumbing.

Reply:
There's a `constants/domNodes.js` (currently just has portal targets, which are mostly unused), might be best to add this id as a constant in there

Reply:
Updated!

Reply:
Looks good!